### DISCUSSION TITLE
Using Sortable, how can I style the placeholder that shows you the preview of where the item is going to be dropped?

### DISCUSSION THREAD

**Discussion by lumenwrites** [clauderic/dnd-kit#929] [Q&A]
In the examples, when you drag a sortable item, it has a half-transparent preview showing where the item is going to drop:

<img width="504" alt="Screen Shot 2022-11-08 at 22 48 57" src="https://user-images.githubusercontent.com/4761235/200692940-eca15a26-f1fe-4742-ac08-86775c3876b3.png">

But when I'm following the [docs](https://docs.dndkit.com/presets/sortable) and trying to implement Sortable in my app, the preview I'm seeing is not transparent:
<img width="894" alt="Screen Shot 2022-11-08 at 22 51 27" src="https://user-images.githubusercontent.com/4761235/200693464-3cffb6fd-234c-4b2e-acd2-8136828e1c8e.png">

Ideally, I'd like to be able to style the preview to make it look like a plain gray version of the card, like so:
<img width="339" alt="Screen Shot 2022-11-08 at 22 56 17" src="https://user-images.githubusercontent.com/4761235/200694066-6aba3c42-7547-4cf2-97b0-77b6f7fd82b7.png">

Can you please help me figure out how to do this?

**Comment by laurent512** [comment]
did you find a solution for that ?

**Comment by brettinternet** [comment]
Hi,

This should answer your question. https://github.com/clauderic/dnd-kit/discussions/446#discussioncomment-1291471
